config.yaml

```diff
@@ -8,6 +8,14 @@
 # - project/config.local.yaml          (global overrides)
 # - project/config.yaml                 (global defaults)
 
+# By default we set the mode to normal.
+# This uses a Chassis box we've built that has Php 7.0, MySQL 5.7 and nginx 1.10.3.
+# Set to base if you'd like to configure Php versions, add extensions and make other modifications.
+# When you set this to base your initial provision will take longer as it will start with the bento/ubuntu-16.04 box.
```

Comment:
* Needs short description on first line.
* These should be line wrapped at 78-80 chars.
* Can we move this way to the bottom instead? We don't want people using it much.

Also, re: "set to base if ...", those can still be set, right?

I'd probably reword this to:
```
Base box mode

By default, we use a box built with the default settings. This speeds up
initial provisioning time and reduces disk usage by sharing the common parts of
the VM.

Set this to `base` to build your own base box.

Values: normal, base
```